### DISCUSSION TITLE
Add deterministic random seed to rdss and rss functions

### DISCUSSION THREAD

**Discussion by jaimeliew1** [python-control/python-control#705] [Q&A]
Is it possible to set a random seed when generating random state space systems using [drss](https://python-control.readthedocs.io/en/0.9.1/generated/control.drss.html#control.drss) and [rss](https://python-control.readthedocs.io/en/0.9.1/generated/control.rss.html#control.rss)? It would be quite helpful to be able to deterministically regenerate the same system, perhaps with an optional `seed` parameter.

**Comment by bnavigator** [comment]
Hi @jaimeliew1,

the current functions use `numpy.random.randn()`, so you can just use the legacy `numpy.random.seed()` in order to get reproducible results:
```
>>> import numpy as np
>>> from control import rss, drss
>>> 
>>> np.random.seed(101)
>>> rss(1,1,3)
StateSpace(array([[-2.4792831]]), array([[ 0.        , -0.31931804, -0.84807698]]), array([[0.60596535]]), array([[-2.01816824,  0.        ,  0.        ]]))
>>> drss(1,1,3)
StateSpace(array([[-0.62606515]]), array([[0.00858304, 0.        , 1.58917569]]), array([[0.53322084]]), array([[ 0.85734924,  0.08707633, -0.        ]]), True)
>>> 
>>> np.random.seed(101)
>>> rss(1,1,3)
StateSpace(array([[-2.4792831]]), array([[ 0.        , -0.31931804, -0.84807698]]), array([[0.60596535]]), array([[-2.01816824,  0.        ,  0.        ]]))
>>> drss(1,1,3)
StateSpace(array([[-0.62606515]]), array([[0.00858304, 0.        , 1.58917569]]), array([[0.53322084]]), array([[ 0.85734924,  0.08707633, -0.        ]]), True)
>>> 
```

**Reply by henklaak** [reply]
Let's add it to the docs (if not already there). Common use case, I'd say.
